src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -2,12 +2,36 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Linq;
 using Microsoft.Data.DataView;
 using Microsoft.ML.Data;
 using Microsoft.ML.Transforms;
 
 namespace Microsoft.ML
 {
+    public sealed class SimpleColumnInfo
```

Comment:
>SimpleColumnInfo [](start = 24, length = 16)

I think @rogancarr has issue where he don't like ColumnInfo name. [https://github.com/dotnet/machinelearning/issues/2554](https://github.com/dotnet/machinelearning/issues/2554)
I wonder how much he would hate this one.

`SimpleMapping`,`SimplePair`?

Reply:
No strong opinion, I just think that we should have a conversation about naming these things :)

---
In reply to: [258285683](https://github.com/dotnet/machinelearning/pull/2641#discussion_r258285683) [](ancestors = 258285683)

Reply:
How about `ColumnNamePair` or something similar? I think something containing 'mapping' or 'pair' would be too general, wouldn't it?

Or I could leave it as `SimpleColumnInfo` and add a comment about it in the issue so we can change it when we find a good name for `ColumnInfo`?

---
In reply to: [258297894](https://github.com/dotnet/machinelearning/pull/2641#discussion_r258297894) [](ancestors = 258297894,258285683)

Reply:
I added mention of this column to #2554.
Let's merge this one, and figure out proper naming in that issue.